src/components/Label/Label.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import {Label as CoreLabel, LabelProps as CoreLabelProps} from 'wix-ui-core/Label';
+import style from './Label.st.css';
+import {withStylable} from 'wix-ui-core/withStylable';
+
+export type Appearance = 'T1.1' | 'T3.1';
+
+export interface LabelProps {
+  /** typography of the label */
+  appearance?: Appearance;
+  children?: string;
+  for?: string;
+  id?: string;
+}
+
+const defaultProps: LabelProps = {
+  appearance: 'T1.1',
```

Comment:
Check the new API for Text: https://wix.github.io/wix-ui-backoffice/?selectedKind=Components&selectedStory=Text&full=0&addons=0&stories=1&panelRight=0

Instead of having `appearance="T1.1"` which means: size="medium" and color=secondary, let's change the API to this `size` and `secondary`. see how it is done in `Text`.

I just want to see the Zeplin to make sure what is the API we need to support